### DISCUSSION TITLE
Tiny version

### DISCUSSION THREAD

**Discussion by Ahycavv** [dream-num/univer#2944] [Q&A | 常见问题]
Hey, I was looking at my page load speed after adding a Univer spreadsheet and noticed that it tanks due to Univer's JS files size. I understand Univer has a lot of features so it has become a big library, so I was wondering if there's a way to only use the basic features. I.e. can I load the spreadsheet editor without loading all plugins mentioned [here](https://univer.ai/guides/sheet/getting-started/installation#usage)? Or is there another way to speed up my page load while using Univer (aside from the obvious file splitting, minification, etc)?

**Comment by jikkai** [comment]
You can consider loading some plugins asynchronously.

**Reply by Ahycavv** [reply]
I see, thanks. I'll try the general web dev solutions for speed improvement and see if the performance is acceptable.
